examples/12_ammonia_synloop/tech_config.yaml

```diff
@@ -41,22 +41,19 @@ technologies:
 
   h2_storage:
     performance_model:
-      model: "h2_storage"
+      model: "storage_auto_sizing"
     control_strategy:
       model: "pass_through_controller"
     cost_model:
-      model: "h2_storage"
+      model: "salt_cavern_cost"
     model_inputs:
-      performance_parameters:
-        electrolyzer_rating_mw_for_h2_storage_sizing: 640
-        size_capacity_from_demand:
-          flag: True # If True, then storage is sized to provide steady-state storage
-        capacity_from_max_on_turbine_storage: False # if True, then days of storage is ignored and storage capacity is based on how much h2 storage fits on the turbines in the plant using Kottenstete 2003.
-        type: "salt_cavern" # can be one of ["none", "pipe", "turbine", "pressure_vessel", "salt_cavern", "lined_rock_cavern"]
-        days: 0 #how many days worth of production we should be able to store (this is ignored if `capacity_from_max_on_turbine_storage` is set to True)
-      control_parameters:
+      shared_parameters:
         commodity_name: "hydrogen"
         commodity_units: "kg/h"
+      cost_parameters:
```

Comment:
I would prefer to have "auto" or something more transparent to the user, but that is probably outside the scope of this PR, maybe more related to #198?

Reply:
I'm talking about in the cost_parameters where the user has to provide initial values that are overridden. If the initial values are just thrown away and have no impact on the final storage capacity I would prefer that the user put in something that is easy for readers to understand as allowing the size to change rather than having the user provide sizes that mean nothing. As is, I would expect many mistakes and a lot of wasted time down the road.

Reply:
I think I'm actually ok with this, but the use case here should be considered in #223. I'm not completely comfortable with the flag and then leaving out the sizing values, but I don't have a verifiably better idea right now.